include/Print.h

```diff
@@ -8,17 +8,72 @@ namespace PySTL
 // TODO [dshvetsov] : Add support of output specification
 //                    (such set_precision, set_base, etc).
 
+namespace detail {
+
+template <typename Head, typename... Args>
+class __print_impl
```

Comment:
Don't use "__" in your variable and class names. Names with "__" reserved for stl.